src/components/VDialog/VDialog.js

```diff
@@ -3,13 +3,22 @@ require('../../stylus/components/_dialogs.styl')
 import Detachable from '../../mixins/detachable'
 import Overlayable from '../../mixins/overlayable'
 import Toggleable from '../../mixins/toggleable'
+import { factory as DependentFactory } from '../../mixins/dependent'
+
+const Dependent = DependentFactory({ closeDependents: true, isDependent: false })
+
+import { factory as StackableFactory } from '../../mixins/stackable'
+
+const Stackable = StackableFactory({ minZIndex: 200, stackClass: 'dialog__content__active' })
 
 import ClickOutside from '../../directives/click-outside'
 
+import { getZIndex } from '../../util/helpers'
```

Comment:
Update this to match this spec https://docs.google.com/document/d/1xWWnAw2zYce3sVvdiRC0OuIhNx1PCt-oCNzTXeREFaw/edit?usp=sharing